source/_config.yml

```diff
@@ -3,9 +3,9 @@
 #######################################################################################
 # Book settings
 
-title: DSCΙ 100
+title: DSCΙ 100 Py
 author: UBC
-copyright: "2021" # Copyright year to be placed in the footer
+copyright: "Since 2022" # Copyright year to be placed in the footer
```

Comment:
remove "since"

Reply:
Removed, although I think we will have to update it every year then, won't we? I usually see something like 2022-23 for example.

Reply:
nope -- specifying a copyright date just refers to when the work was first created. it does not need to be updated

(honestly specifying a copyright date is pointless anyway -- copyright law in basically any jurisdiction applies from the first provable date of creation anyway, regardless of notices posted)